### ISSUE TITLE
请问有人将crnn_lite.py 转成keras模型吗?求指教

### ISSUE THREAD

**Issue by simmerken** [ouyanghuiyu/chineseocr_lite#148]
cnn.add_module('conv{0}'.format(i),
                               nn.Conv2d( nIn,  nIn, ks[i], ss[i], ps[i],groups=nIn)

这句出来的keras的权重尺寸转不正确, 求指教.

keras 我是这么改的:
     x = Conv2D(filters=nIn,
 49                    kernel_size=ks[i],
 50                    strides=(ss[i], ss[i]),
 51                    padding= 'valid' if ps[i]==0 else 'same',
 52                    dilation_rate=(1, 1),                                  
 53                    activation=None, use_bias=True,data_format=data_format,
 54                    name='cnn.conv{0}'.format(i)
 55                    )(x)
